sos/report/plugins/nvidia.py

```diff
@@ -30,8 +32,15 @@ def setup(self):
             'nvlink -e'
         ]
 
-        self.add_service_status("nvidia-persistenced")
+        ctk_subcmds = [
+            'cdi list',
+            '--version',
+        ]
+
+        self.add_copy_spec("/etc/cdi/nvidia.yaml")
```

Comment:
Just a nitpick, for the sake of code/plugin readiness, I would move the `a_s_c` to the beginning of `setup(self)` - or at least outside of the logical block:

    subcmds = ..
    ctk_subcmds = ..
    # something completely else here..?
    add_cmd_output( .. subcmds ..)
    add_cmd_output( .. ctk_subcmds ..)

Reply:
Makes sense, I'll do that